### ISSUE TITLE
New release crashing?

### ISSUE THREAD

**Issue by njt1982** [WyriHaximus/github-action-composer.lock-diff#157]
Hi,

Not sure if its related to the 2.0.2 release from 3 hours ago... but our action has just stopped working and there is no obvious error...

```
##[debug]0
##[debug]commit a5d465c303076f2c87556f83fc04ca3d7b5b2404
##[debug]Author: Nicholas Thompson <nick@fullfatthings.com>
##[debug]Date:   Fri Apr 26 15:26:59 2024 +0100
##[debug]
##[debug]    XXXXXXXXXXX
##[debug]
/usr/bin/git log -1 --format='%H'
'a5d465c303076f2c87556f83fc04ca3d7b5b2404'
##[debug]0
##[debug]'a5d465c303076f2c87556f83fc04ca3d7b5b2404'
##[debug]
##[debug]Unsetting HOME override
::remove-matcher owner=checkout-git::
##[debug]Removed matchers: 'checkout-git'
##[debug]Node Action run completed with exit code 0
##[debug]Save intra-action state isPost = true
##[debug]Save intra-action state setSafeDirectory = true
##[debug]Save intra-action state repositoryPath = /home/runner/work/XXXXXX/.wyrihaximus-composer.lock-diff/checkout/base-ref
##[debug]Finished: run
##[debug]Evaluating condition for step: 'run'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: run
##[debug]Loading inputs
##[debug]Evaluating: inputs.dryRun
##[debug]Evaluating Index:
##[debug]..Evaluating inputs:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'dryRun'
##[debug]=> 'no'
##[debug]Result: 'no'
##[debug]Evaluating: inputs.workingDirectory
##[debug]Evaluating Index:
##[debug]..Evaluating inputs:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'workingDirectory'
##[debug]=> ''
##[debug]Result: ''
##[debug]Loading env
Run docker://ghcr.io/wyrihaximus/github-action-composer.lock-diff:v2
  with:
    dryRun: no
  env:
    GITHUB_TOKEN: ***
/usr/bin/docker run --name ghcriowyrihaximusgithubactioncomposerlockdiffv2_08eb5d --label 8d3b1d --workdir /github/workspace --rm -e "GITHUB_TOKEN" -e "INPUT_DRYRUN" -e "INPUT_WORKINGDIRECTORY" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_REPOSITORY_OWNER_ID" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_REPOSITORY_ID" -e "GITHUB_ACTOR_ID" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKFLOW_REF" -e "GITHUB_WORKFLOW_SHA" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e "GITHUB_ACTION_PATH" -e "RUNNER_DEBUG" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_ENVIRONMENT" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e "ACTIONS_RESULTS_URL" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/XXXXX":"/github/workspace" ghcr.io/wyrihaximus/github-action-composer.lock-diff:v2
Working Directory:

##[debug]Docker Action run completed with exit code 255
##[debug]Finished: run
##[debug]Finishing: Run WyriHaximus/github-action-composer.lock-diff@v2
```

Anything else I can provide to help debug? The action is:

```
name: Checks
on: pull_request
jobs:
  composer-lock-diff:
    name: Composer lock diff
    runs-on: ubuntu-latest
    permissions:
      pull-requests: write
      contents: read
    steps:
      - uses: actions/checkout@v2
      - uses: WyriHaximus/github-action-composer.lock-diff@v2
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by longwave** [comment]
```
Fatal error: Declaration of IonBazan\ComposerDiff\Command\DiffCommand::execute(Symfony\Component\Console\Input\InputInterface $input, Symfony\Component\Console\Output\OutputInterface $output) must be compatible with Symfony\Component\Console\Command\Command::execute(Symfony\Component\Console\Input\InputInterface $input, Symfony\Component\Console\Output\OutputInterface $output): int in /workdir/vendor/ion-bazan/composer-diff/src/Command/DiffCommand.php on line 127
```

ion-bazan/composer-diff needs upgrading to 1.8 or later: https://github.com/IonBazan/composer-diff/releases/tag/v1.8.0

**Comment by WyriHaximus** [comment]
Just released [`v2.0.3`](https://github.com/WyriHaximus/github-action-composer.lock-diff/releases/tag/v2.0.3) with your fix, thank you for that 👍

**Comment by WyriHaximus** [comment]
Sorry for the disruption, will continue working on a fix for the issue you reported @njt1982.
